src/online2/online-nnet2-feature-pipeline.h

```diff
@@ -104,10 +108,12 @@ struct OnlineNnet2FeaturePipelineConfig {
     opts->Register("fbank-config", &fbank_config, "Configuration file for "
                    "filterbank features (e.g. conf/fbank.conf)");
     opts->Register("cmvn-config", &cmvn_config, "Configuration file for "
```

Comment:
At the suggestions of @danpovey, we changed this to feat-cmvn-config

Reply:
Hm.  If this setup is already tested, I might be OK to have the name stay as it is.  I was not 100% sure/confident about that change actualy.

Reply:
Yes, I would prefer to keep it as it is too (CMVN is always applied to features, so the word 'feat' does not bring more information)

Reply:
Great by me